_posts/2021-04-26-changelog-146.markdown

```diff
@@ -0,0 +1,190 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #146"
+date: 2021-04-26 17:00:00 +0300
+---
+
+
+## New Features
+
+* [#6992] Provide initial experimental support for [custom derive]
+  procedural macros. Now the plugin can expand such procedural macro calls and take into account expanded `impl` items
+  in type inference and name resolution (other types of items are ignored for now).<br/>
+  Note, it’s only an initial implementation,
+  so it may work in an unexpected way in some cases. The feature is disabled by default for now. To turn it on, you
+  should enable `Use experimental name resolution engine` option in `Preferences | Languages & Frameworks | Rust`
+  settings and enable `org.rust.cargo.evaluate.build.scripts` and `org.rust.macros.proc` experimental features
+  (see instructions [here][experimental features]).
+  Don’t forget to reload a project structure after enabling the corresponding features via `Refresh Cargo Projects`
+  action on Cargo tool window. See [tracking issue](https://github.com/intellij-rust/intellij-rust/issues/6908) for
+  more details and the current status of procedural macros support
+
+  {% include gif-img-wh.html path="/assets/posts/changelog-146/custom-derive" w="600px" h="300px" %}
```

Comment:
Do you actually need to specify both `width` and `height`? It seems it's quite easily to write wrong ratio to make something like
![image](https://user-images.githubusercontent.com/2539310/116001301-76e51b80-a5fc-11eb-9b0b-13b9d35f39b4.png)
Maybe `height` is enough here?